scripts/updateUsageDocs.js

```diff
@@ -0,0 +1,26 @@
+const { writeFile, readFile } = require('fs').promises;
```

Comment:
`/* eslint-disable @typescript-eslint/no-var-requires */` needs to be added in first line